pkg/build/reaper/reaper_test.go

```diff
@@ -0,0 +1,195 @@
+package reaper
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kerrors "k8s.io/kubernetes/pkg/api/errors"
+	ktestclient "k8s.io/kubernetes/pkg/client/unversioned/testclient"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	buildutil "github.com/openshift/origin/pkg/build/util"
+	"github.com/openshift/origin/pkg/client/testclient"
+)
+
+func mkbuildconfig(version int) *buildapi.BuildConfig {
```

Comment:
I don't like this naming scheme. Please stick with what we have mostly, iow. `makeBuildConfig`. Yes, I'm aware the deployment reaper looks like that ;)